composer.json

```diff
@@ -56,7 +56,5 @@
     "branch-alias": {
       "dev-feature/contao4": "4.0.x-dev"
     }
-  },
-  "prefer-stable": true,
-  "minimum-stability": "dev"
```

Comment:
I have add minimum stability so we can install this package as `"contao-community-alliance/events-contao-bindings": "~4.0"`. After this package is a stable version, would i remove the minimum stability.

@discordier What do you mean